alerts/ui/package.json

```diff
@@ -47,6 +48,7 @@
     "tailwindcss": "^3.3.1",
     "url-state-provider": "https://assets.juno.global.cloud.sap/libs/url-state-provider@1.3.2/package.tgz",
     "util": "^0.12.4",
+    "utils": "https://assets.juno.global.cloud.sap/libs/utils@latest/package.tgz",
```

Comment:
Could you please use a specific version instead of "latest"? Using "latest" can lead to integrity check issues when the SHA changes between different versions.